src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1775,7 +1775,12 @@ private function export($value)
 
     private function doExport($value, $resolveEnv = false)
     {
-        if (is_string($value) && false !== strpos($value, "\n")) {
+        if (is_string($value) && \in_array(preg_match('/[^\s\P{Cc}]/u', $value), [false, 1], true)) {
```

Comment:
Is the change for the PHP dumper really necessary? Was there anything breaking before?

Reply:
Not really but then generated file could contain non-utf8 characters and opening generated file with an ide leads to warnings about file encoding.

Reply:
nothing we need to worry about IMO